1.7/usage/service-discovery/virtual-ip-addresses.md

```diff
@@ -24,11 +33,11 @@ To create a VIP:
 
     **Tip:** Select **JSON Mode** to edit your application directly in JSON.
 
-    For more information on port configuration, see the [ports documentation][1].
+    For more information on port configuration, see the [ports documentation][2].
 
 ## Assigning Multiple VIPs to Your App
 
-To assign multiple VIPs to your application, switch to JSON mode and add the additional VIPs to your `portDefinitions`. In this example, the additional VIP added is `"VIP_1": "111.2.1.23:5050"`:
+To assign multiple VIPs to your application, switch to JSON mode and add the additional VIPs to your `portDefinitions`. In this example, the additional VIP added is `"VIP_1": "192.168.0.1:5050"`:
```

Comment:
`172.0.0.0` isn't within the private range, but `172.16.0.0/12` is :)

Updated accordingly, and thanks for the suggestion.